### DISCUSSION TITLE
How to jump to another note (internal link) from inside a canvas note?

### DISCUSSION THREAD

**Discussion by markbj71** [TriliumNext/Trilium#4190] [Q&A] [answered]
The Excalidraw support (canvas note) is great, but is it possible to jump to another note from inside a canvas note?

In the canvas note, we can add a standard web link to a shape, I can put the link as "https://somedomain/#root/id-path-to-anothernode"
but by clicking it, we will see a new browser window. I tried with "#root/id-path-to-anothernode" but it doesn't work.

Could we just have an internal link inside a canvas note?

**Comment by meichthys** [comment, marked as answer]
This is an open issue here: https://github.com/zadam/trilium/issues/2960

**Reply by markbj71** [reply]
Thanks!
